san/sanbase_graphql.py

```diff
@@ -95,25 +95,12 @@
         'query': 'miningPoolsDistribution',
         'return_fields': ['datetime', 'other', 'top10', 'top3']
     },
-    'historical_balance':{
-        'query': 'historicalBalance',
-        'return_fields': ['balance', 'datetime']
-    },
-    'social_dominance':{
-        'query': 'socialDominance',
-        'return_fields': ['datetime', 'dominance']
-    },
-    'top_holders_percent_of_total_supply':{
-        'query': 'topHoldersPercentOfTotalSupply',
-        'return_fields': ['datetime', 'inExchanges', 'inTopHoldersTotal', 'outsideExchanges']
-    },
```

Comment:
Why are these deleted?

Reply:
Because they're not "equal format" functions, that means that they're not created via "_create_query_string", which automates the process and grabs the data for making functions from the QUERY_MAPPING